code/main/task/eventProcessor.brs

```diff
@@ -20,11 +20,13 @@ function _adb_EventProcessor(task as object) as object
         _configurationModule: invalid,
         _identityModule: invalid,
         _edgeModule: invalid,
+        _mediaModule: invalid,
 
         init: function() as void
             m._configurationModule = _adb_ConfigurationModule()
             m._identityModule = _adb_IdentityModule(m._configurationModule)
             m._edgeModule = _adb_EdgeModule(m._configurationModule, m._identityModule)
+            m._mediaModule = _adb_MediaModule(m._configurationModule, m._identityModule)
```

Comment:
Can we pass the edgeModule to mediaModule and in the media module and after processing the media hit forward it to edgeModule like we do here in sendEvent() function? I want to explore if this is a cleaner way over the current approach.

Reply:
It's intended to let the Media module hold an Edge request queue and handle the Media Edge requests/responses. I was thinking of using the single Edge request queue in the Edge module. But it runs into the complicated code logic when handling the queued requests/responses for both Edge and Media modules. I'm open to other options.

Reply:
Discuss it offline, we still use two queues for processing edge and media edge requests. And we decided to refactor the Media module to remove external dependencies to the identity and edge modules.